packages/smooth_app/pubspec.lock

```diff
@@ -1499,5 +1492,5 @@ packages:
     source: hosted
     version: "3.1.1"
 sdks:
-  dart: ">=2.17.0 <3.0.0"
-  flutter: ">=3.0.0"
+  dart: ">=2.18.0 <3.0.0"
+  flutter: ">=3.1.0-0"
```

Comment:
That's a version to high, please revert the changes in the pubspec.lock or just regenerate With 3.0.5

Reply:
Yeah I will just commit the read me file , dono why pubspec file got updated